### DISCUSSION TITLE
FUSE application and ctrl+c combination

### DISCUSSION THREAD

**Discussion by parrado** [dokan-dev/dokany#1035] [Q&A] [answered]
I've ported a FUSE application to use dokany which needs to be executed with admin rights. However, ctrl+c combination only works if -d flag is used, otherwise application won't respond to ctrl+c. 

Is there any way to make ctrl+c working again with no need of -d flag?. Thanks in advance.

**Comment by Liryna** [comment, marked as answer]
`-d` option is for debugging. It will make the process not be daemonized (keep attached to the console) which is needed to catch ctrl + c. You also have the option `-f` that does the same.

**Reply by parrado** [reply]
Thanks,
